metadata-ingestion/src/datahub/ingestion/source/schema_inference/csv_tsv.py

```diff
@@ -0,0 +1,74 @@
+from typing import IO, Dict, List, Type
+
+from tableschema import Table
+
+from datahub.ingestion.source.schema_inference.base import SchemaInferenceBase
+from datahub.metadata.com.linkedin.pegasus2avro.schema import (
+    ArrayTypeClass,
+    BooleanTypeClass,
+    DateTypeClass,
+    NullTypeClass,
+    NumberTypeClass,
+    RecordTypeClass,
+    SchemaField,
+    SchemaFieldDataType,
+    StringTypeClass,
+    TimeTypeClass,
+    UnionTypeClass,
+)
+
+# see https://github.com/frictionlessdata/tableschema-py/blob/main/tableschema/schema.py#L545
+tableschema_type_map: Dict[str, Type] = {
+    "duration": TimeTypeClass,
+    "geojson": RecordTypeClass,
```

Comment:
Does DataHub have support for these types?
Can we use a library like https://geoalchemy-2.readthedocs.io/en/latest/ instead of hard coding this?

Reply:
As far as I know we don't have support for GIS types — I'm not sure how geoalchemy would solve this since it's supposed to help with SQLAlchemy rather than this TableSchema package.